.github/SECURITY.md

```diff
@@ -14,8 +14,7 @@ Security updates will typically only be applied to the latest release (at least
 
 ## Reporting a vulnerability
 
-To report a security issue, email [security@jans.io](mailto:security@jans.io?subject=SECURITY)
-and include the word "SECURITY" in the subject line.
+To report a security issue, send an email to [security@jans.io](mailto:security@jans.io?subject=SECURITY)
```

Comment:
is this email working ?

Reply:
It looks like this is from a find/replace for gluu.org -> jans.io .

Reply:
Oh no no. Not a find/replace. 
@ganesh-at-wiw had created this email long back on my request. And as far as I can remember, we had tested it as well. I have asked @ganesh-at-wiw to confirm if it is still working.

Reply:
Tested again. Mail ID is working.